tests/worker/test_task_manager.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+from collections.abc import AsyncIterator
+
+import pytest
+
+from saturn_engine.worker.task_manager import TaskManager
+
+
+@pytest.fixture
+async def task_manager() -> AsyncIterator[TaskManager]:
+    _task_manager = TaskManager()
+    yield _task_manager
+    await _task_manager.close()
+
+
+@pytest.mark.asyncio
+async def test_task_manager(  # noqa: C901  # Ignore complexity errors.
```

Comment:
ignore it globally?

Reply:
This can be useful to trigger function split. I don't mind keeping it.